src/client/eppo-client.ts

```diff
@@ -501,6 +501,51 @@ export default class EppoClient {
     return { variation, action };
   }
 
+  /**
+   * Evaluates the supplied actions and returns the best ranked action.
+   *
+   * This method should be considered **preview** and is subject to change as requirements mature.
+   *
+   * CAUTION: This method does not do any logging or assignment computation and so calling this method will have
```

Comment:
How about instead of "caution" we do "Note:" (seems less aggressive?)

Reply:
made some changes to lessen chance devs use this method inappropriately